templates/jws57-openjdk17-tomcat9-ubi8-https-s2i.json

```diff
@@ -54,7 +54,7 @@
             "displayName": "Git Reference",
             "description": "Git branch/tag reference",
             "name": "SOURCE_REPOSITORY_REF",
-            "value": "main",
+            "value": "1.2",
```

Comment:
This should remain unchanged, with the value of "main", as the [repo](https://github.com/web-servers/tomcat-websocket-chat-quickstart) the template references doesn't have a branch or tag named "1.2".